### ISSUE TITLE
NTR: 2-methyl-2,3-dihydroxybutyrate high in urine (Lab: chemistry: organic acids)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8643]
New term request 2-methyl-2,3-dihydroxybutyrate high in urine (elthat_190815153634): 2-methyl-2,3-dihydroxybutyric acid

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000469
- label: Elevated urine 2,3-dihydroxy-2-methylbutanoic acid level
- parents: [Monocarboxylic aciduria (HP:0034653)](https://hpo.jax.org/app/browse/term/HP:0034653)
- pmid: PMID:29575569
- def: The amount of 2,3-dihydroxy-2-methylbutanoic acidin the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8643
- synonyms: Elevated urine 2-methyl-2,3-dihydroxybutyrate level(NONE)|2-methyl-2,3-dihydroxybutyrate high in urine(NONE)
